Zend/zend_attributes.h

```diff
@@ -7,13 +26,15 @@
 #define ZEND_ATTRIBUTE_TARGET_PROPERTY		(1<<3)
 #define ZEND_ATTRIBUTE_TARGET_CLASS_CONST	(1<<4)
 #define ZEND_ATTRIBUTE_TARGET_PARAMETER		(1<<5)
-#define ZEND_ATTRIBUTE_TARGET_ALL			(1<<6)
+#define ZEND_ATTRIBUTE_TARGET_ALL			((1<<6) - 1)
+#define ZEND_ATTRIBUTE_IS_REPEATABLE		(1<<6)
+#define ZEND_ATTRIBUTE_FLAGS				((1<<7) - 1)
 
-#define ZEND_ATTRIBUTE_SIZE(argc) (sizeof(zend_attribute) + sizeof(zval) * (argc) - sizeof(zval))
+#define ZEND_ATTRIBUTE_SIZE(argc) (sizeof(zend_attribute) + sizeof(zval) * (argc))
```

Comment:
Why this change?

Reply:
An unintended leftover from searching for a segfault. I restored it to the previous version.